app/views/dashboard/districts/show.html.erb

```diff
@@ -243,96 +243,87 @@
+          <div class="mb-1 mb-lg-0 pt-lg-3 pb-lg-4 flex-lg-2 align-lg-center justify-lg-center">
             <p class="mb-0 ls-145">
-              Q2 cohort BP control
+              Control rate of all HTN patients
             </p>
           </div>
-          <div class="mb-0 mb-lg-0 pt-lg-4 pb-lg-4 flex-lg-1 align-lg-center justify-lg-center br-lg br-lg-grey-light">
+          <div class="mb-0 mb-lg-0 pt-lg-3 pb-lg-4 flex-lg-1 align-lg-center justify-lg-center">
             <p class="mb-0 ta-lg-center c-grey-dark">
-              <span class="fw-bold d-lg-none">Bathinda: </span> Coming Soon...
+              <span class="fw-bold d-lg-none">
+                Top District: 
+              </span>
+              <span>
+                <%= number_to_percentage(
+                   @top_district_benchmarks[:controlled_percentage],
+                   precision: 0
+                ) %>
+              </span>
             </p>
           </div>
-          <div class="mb-0 mb-lg-0 pt-lg-4 pb-lg-4 flex-lg-1 align-lg-center justify-lg-center br-lg br-lg-grey-light">
+          <div class="mb-0 mb-lg-0 pt-lg-3 pb-lg-4 flex-lg-1 align-lg-center justify-lg-center">
             <p class="mb-0 ta-lg-center c-grey-dark">
-              <span class="fw-bold d-lg-none">Top district: </span> Coming Soon...
+              <span class="fw-bold d-lg-none">
+                <%= @district.name %>: 
+              </span>
+              <span>
+                <% control_rate = compute_percentage(@controlled_patients.values.last.to_f, @registrations.values.last.to_f) %>
+                <%= control_rate %>
+              </span>
             </p>
           </div>
-          <div class="mb-0 mb-lg-0 pt-lg-4 pb-lg-4 flex-lg-1 align-lg-center justify-lg-center">
+          <div class="mb-0 mb-lg-0 pt-lg-3 pb-lg-4 flex-lg-1 align-lg-center justify-lg-center">
             <p class="mb-0 ta-lg-center c-grey-dark">
-              <span class="fw-bold d-lg-none">Difference: </span> Coming Soon...
+              <span class="fw-bold d-lg-none">
+                Difference: 
+              </span>
+              <% controlled_percentage = (control_rate.to_f - @top_district_benchmarks[:controlled_percentage]).round %>
+              <span class="<%= controlled_percentage > 0 ? "c-green-dark" : "c-red" %>">
+                <%= controlled_percentage > 0 ? "+" : "" %><%= controlled_percentage %>%
+              </span>
             </p>
           </div>
         </div>
-        <div class="d-lg-flex mb-lg-0 pb-lg-0">
-          <div class="mb-1 mb-lg-0 pt-lg-4 pb-lg-4 flex-lg-2 align-lg-center justify-lg-center br-lg br-lg-grey-light">
-            <p class="mb-0 ls-145">
-              Q2 cohort missed visit
+        <div class="mb-3 pb-3 d-lg-flex mb-lg-2 pb-lg-0">
+          <div class="mb-1 mb-lg-0 pt-lg-3 pb-lg-4 flex-lg-2 align-lg-center justify-lg-center">
+            <p class="mb-0 ls-145 c-grey-dark">
+              More benchmarks TBD
```

Comment:
```suggestion
              More benchmarks coming soon
```

Reply:
Oh ya, this will get resolved in the next PR to land

Reply:
er maybe not, i'll just merge it here